cassandra-commons/src/main/java/com/mesosphere/dcos/cassandra/common/tasks/CassandraTaskExecutor.java

```diff
@@ -254,6 +256,8 @@ public CassandraTaskExecutor update(final ExecutorConfig config) {
                         config.getArguments(),
                         config.getURIs(),
                         ImmutableMap.<String, String>builder()
+                                .put("LD_LIBRARY_PATH", "$MESOS_SANDBOX/libmesos-bundle/lib")
```

Comment:
I wouldn't expect dynamic `LD_LIBRARY_PATH`/`MESOS_NATIVE_JAVA_LIBRARY` to work as envvars. Shouldn't they be `export`s in the command itself?